src/components/global/BaseButton.vue

```diff
@@ -5,12 +5,10 @@ export default defineComponent({
   name: 'BaseButton',
   props: {
     type: {
-      type: String,
-      default: 'button'
+      type: String
     },
     to: {
-      type: Object,
-      default: { name: 'Home' }
+      type: Object
```

Comment:
It *could* be a string sometimes (even though it's not recommended to use paths as strings on our router links), so just to be safe... 😄 
```suggestion
      type: [String, Object]
```

Reply:
Of course, really need to keep in mind that others might use my components - and in a different way than I can imagine :alien: